wdl2cwl/tests/test_wdl_stdlib.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for the translation of WDL Stdlib functions."""
+
+from tempfile import NamedTemporaryFile
+
+from wdl2cwl.main import convert
+
+
+def test_select_all() -> None:
+    """Tests the ``select_all`` function."""
+    # Example based off miniwdl's test_select test function:
+    # https://github.com/chanzuckerberg/miniwdl/blob/0ee2bddaea4bd2538cd21bd9b7806a6c81068b2f/tests/test_5stdlib.py#L156
+    wdl = R"""
+    version 1.0
+    task test_select_all {
+      input {
+        Int one
+        Int? two
+      }
+      command {}
+      output {
+        Array[Int] first1 = select_all([one, two])
+      }
+    }
+    """
+    with NamedTemporaryFile(mode="w", encoding="utf-8") as output:
```

Comment:
@mr-c I wrote this one reading the code [^1] & tests [^2] for `select_all` in miniwdl. I didn't have time to search for a workflow to write the functional test, so quickly wrote a unit test. Do you think it'd be a good idea to add tests like this one? Or do you think it'd be better to only use the functional tests with a given WDL file, and another CWL file with the expected result?

[^1]: https://github.com/chanzuckerberg/miniwdl/blob/0ee2bddaea4bd2538cd21bd9b7806a6c81068b2f/WDL/StdLib.py#L769
[^2]: https://github.com/chanzuckerberg/miniwdl/blob/0ee2bddaea4bd2538cd21bd9b7806a6c81068b2f/tests/test_5stdlib.py#L156